src/AvatarButton/index.d.ts

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+export interface AvatarButtonProps {
+  alt?: string,
+  size?: 'xs' | 'sm' | 'md' | 'lg' | 'xl' | 'xxl' | 'huge',
```

Comment:
I think this sizes it`s a good Union for `Avatar` and `AvatarButton` components. 
Maybe, we will create an `AvatarSizesVariants` union, and we will use in these couple components?